### PR TITLE
add eventsPerPage to schedule

### PR DESCRIPTION
**Pull request by alex-Arc**
image
image

### CODE REVIEW

File: apps/client/src/common/components/view-params-editor/constants.ts [hunk 1]
```diff
@@ -224,6 +230,12 @@ export const PUBLIC_OPTIONS: ParamField[] = [
     description: 'Schedule will not auto-cycle through events',
     type: 'boolean',
   },
+  {
+    id: 'eventsPerPage',
+    title: 'Number of events on page',
+    description: '',
```
Comment (cpvalente) [inline apps/client/src/common/components/view-params-editor/constants.ts:236]: Just for sake of consistency we can use the title and description fields
The description field can also warn users that the events will overflow the page

**Review by cpvalente — COMMENTED** [review]
If the user adds too many events in a column, they will see some overflow. What does this look like? could we add some styling to make sure it doesn't look terrible?

Why only on the Public and not in the Backstage screens?

**Comment by alex-Arc** [comment]
I tied looking in to some overflow effects but it is beyond my ability
https://amorriscode.github.io/use-overflow/

the clipping doesn't look to bad 😉 
image

**Review by cpvalente — APPROVED** [review]
Looks good, thank you!

**Pull request merged**